platform/ui/src/components/DisplaySetMessageListTooltip/DisplaySetMessageListTooltip.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Tooltip from '../Tooltip';
+import Icon from '../Icon';
+import { DisplaySetMessage } from '@ohif/core';
+
+const DisplaySetMessageListTooltip = ({ messages }): React.ReactNode => {
+  if (messages.size()) {
+    return (
+      <div>
+        <Tooltip
+          position="left"
+          tight={true}
+          content={
+            <div className="text-left max-w-40">
+              <div className="break-normal text-base text-blue-300 font-bold">
+                Series Messages
+              </div>
+              <ol>
+                {messages.messages.map((message, index) => (
+                  <li key={index}>
+                    {index + 1}. {message.toString()}
+                  </li>
+                ))}
+              </ol>
+            </div>
+          }
+        >
+          <Icon
+            name={
+              messages.includes([DisplaySetMessage.CODES.NOT_RECONSTRUCTABLE])
```

Comment:
In my opinion, I find it a bit odd that the thumbnail decides whether a message is an error or warning. Perhaps we should consider if the message itself has a field that dictates this and can be set when the message is created.

Reply:
This was not required so I removed the if so it will always be warnings